src/app/app.component.html

```diff
@@ -21,59 +21,49 @@
   </button>
 
   <ng-container *ngIf="pageTitle$ | async as pageTitleData">
-        <span>
-          {{ pageTitleData.title | translate }}
-        </span>
-<!--    <app-bookmark-button *ngIf="pageTitleData.isWorkflow"></app-bookmark-button>-->
+      <span>
+        {{ pageTitleData.title | translate }}
+      </span>
+    <!--<app-bookmark-button *ngIf="pageTitleData.isWorkflow"></app-bookmark-button>-->
```

Comment:
After filtering out the whitespace changes, the real change in this file was your great work making the tests more reliable by adding the Cypress `toolbar-setting-button` data attribute.
The changes were introduced [in 6fff9525](https://github.com/kendraio/kendraio-app/commit/6fff9525#diff-8d554c98f4c32fee762d6a18e8e189f8898124312ba390bdc51fadd38345674fL27-R35).

Regarding deletion of the bookmark button comment: the comment provides historical context for the feature and removing it would make that context harder to find. Since the reason for commenting it out is unclear, and unrelated to the upgrade, I suggest that we leave it as it was. Leaving comments alone that are unrelated to the upgrade can help us avoid increasing the complexity of the changes.
So instead of deleting the comments, I have now reverted all of the unrelated changes for this file, the only change is the attribute addition.
If you look at [the file changes now](https://github.com/kendraio/kendraio-app/pull/500/files#diff-8d554c98f4c32fee762d6a18e8e189f8898124312ba390bdc51fadd38345674f), you should see only the intended change, which reduces the diff by 22 lines!

Is that fair?
@gsambrotta